src/components/LottieComponent.jsx

```diff
@@ -0,0 +1,11 @@
+import Lottie from "lottie-react";
+import QrA from "../assets/animations/QRcode-animation.json";
+import LoaderA from "../assets/animations/loader-animation.json";
+
+export const QrAnimation = () => {
+    return <Lottie animationData={QrA} />;
+};
+
+export const LoadingAnimation = () => {
+    return <Lottie animationData={LoaderA} />;
+};
```

Comment:
Fun to see that you experimented a little bit more with animations in this project! :) They are always a nice addition 🥰